### ISSUE TITLE
Where can I get the checkpoint file?

### ISSUE THREAD

**Issue by skipper2cic** [TannerGilbert/Tensorflow-Object-Detection-API-train-custom-Mask-R-CNN-model#17]
Thank you for the tutorial.

Where can I get the pretrained checkpoint file, ckpt-0 ?
Can you share the file too?

**Comment by skipper2cic** [comment]
I've downloaded from the model-zoo.
